### ISSUE TITLE
[Question]: Using Gradio with Workflows

### ISSUE THREAD

**Issue by BlueKiji77** [run-llama/llama_index#16759]
### Question Validation

- [X] I have searched both the documentation and discord for an answer.

### Question

How do I use Workflow event driven structure in with Gradio? Is it possible or should I consider another way for my app frontend?

**Comment by brycecf** [comment]
Yes, it’s possible, but it may be easier with something like Chainlit.

**Comment by BlueKiji77** [comment]
thank you, I will check out Chainlit.

> Yes, it’s possible, but it may be easier with something like Chainlit.
